src/main/java/com/flab/marketgola/common/exception/ApiError.java

```diff
@@ -0,0 +1,37 @@
+package com.flab.marketgola.common.exception;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.http.HttpStatus;
+
+@Getter
+@Setter
+@JsonInclude(Include.NON_EMPTY)
+public class ApiError {
+
+    private final LocalDateTime timestamp;
+    private final HttpStatus status;
+    private final String message;
+    private final String path;
```

Comment:
FE에서 Api에러의 메시지만 보고 분기처리하긴 쉽지 않을 것 같습니다. 서버에서 error code같은 것을 만들어서 내려주면 FE에서 그 코드를 기반으로 예외 처리를 할 수 있을 겁니다. 

예를 들면 이런 거죠. 
DuplicateUser -> 이게 코드가 될 수 있습니다. 
DUPLICATE_USER -> 이것도 코드가 될 수 있죠. 
하지만, duplicate user because .... -> 만약 이런 메시지들은 FE에서 분기하기 힘듭니다. 

때문에 errorCode 등을 만들어서 관리할 필요가 있을 것 같습니다. 

이건 보통 예외 발생 시 전달해주는 정보입니다.